src/latte/functional/disentanglement/mutual_info.py

```diff
@@ -328,6 +339,12 @@ def xmig(
     """
     Calculate Dependency-Blind Mutual Information Gap (XMIG) between latent vectors and attributes
 
+    Dependency-blind Mutual Information Gap (XMIG) is a complementary metric to MIG and DMIG that measures the gap in mutual information with the subtrahend restricted to dimensions which do not regularize any attribute. XMIG is given by
+
+    .. math:: \operatorname{XMIG}(a_i, \mathbf{z}) = \dfrac{\mathcal{I}(a_i, z_j)-\mathcal{I}(a_i, z_k)}{\mathcal{H}(a_i)},
+
+    where :math:`j=\operatorname{arg}\max_d \mathcal{I}(a_i, z_d)`, :math:`k=\operatorname{arg}\max_{d\not\in\mathcal{D}} \mathcal{I}(a_i, z_d)`, 
```

Comment:
Mutual info and entropy need to be defined again for each function. Each function doc should be as standalone as possible.